actors/builtin/miner/deadlines.go

```diff
@@ -153,8 +153,8 @@ func ComputePartitionsSectors(d *Deadlines, partitionSize uint64, deadlineIndex
 
 	// Work out which sector numbers the partitions correspond to.
 	deadlineSectors := d.Due[deadlineIndex]
-	var partitionsSectors []*abi.BitField
-	for _, pIdx := range partitions {
+	partitionsSectors := make([]*abi.BitField, len(partitions))
```

Comment:
I noticed we tend to avoid pre-allocating. Is this a style thing?

Reply:
No, thanks for improving it.